claat/nodes/infobox_test.go

```diff
@@ -0,0 +1,109 @@
+package nodes
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+// AllowUnexported option for cmp to make sure we can diff properly.
+var cmpOpt = cmp.AllowUnexported(InfoboxNode{}, node{}, ListNode{}, TextNode{})
+
+func TestNewInfoboxNode(t *testing.T) {
+	tests := []struct {
+		name      string
+		inKind    InfoboxKind
+		inContent []Node
+		out       *InfoboxNode
+	}{
+		{
+			name:   "PositiveEmpty",
+			inKind: InfoboxPositive,
+			out: &InfoboxNode{
+				node: node{typ: NodeInfobox},
+				Kind: InfoboxPositive,
+				// TODO: Do we really want this to not be nil?
```

Comment:
I think it's acceptable as is because we're calling the NewListNode's empty method when we evaluate empty.

Reply:
We could just as easily check if Content is nil, or perhaps in addition to. Would like to defer this to a separate PR

Reply:
SGTM.